cover.html

```diff
@@ -0,0 +1,1988 @@
+
```

Comment:
Can we remove this?

Reply:
Will do, I thought I added it to gitignore